assets/src/application/ui/layout/entityList/filterBar/buttons/ToggleFiltersButton.tsx

```diff
@@ -8,10 +8,10 @@ import { LabelPosition } from '@application/ui/display';
 import { ToggleFiltersButtonProps } from '../types';
 import { getPropsAreEqual } from '@appServices/utilities';
 
-const ToggleFiltersButton: React.FC<ToggleFiltersButtonProps> = React.memo(({ listId, showFilters, toggleFilters }) => {
+const ToggleFiltersButton: React.FC<ToggleFiltersButtonProps> = ({ listId, showFilters, toggleFilters, ...rest }) => {
 	const className = classNames('ee-filter-bar__btn', { 'ee-filter-bar__btn--active': showFilters });
 	const filterId = `ee-toggle-filters-btn-${listId}`;
-	const tooltip = __(`${showFilters ? 'hide' : 'show'} filters`);
+	const tooltip = showFilters ? __('hide filters') : __('show filters');
```

Comment:
As mentioned earlier, gettext functions cannot handle variables.

Reply:
I know who created that variable :)

Reply:
no, it was me:
![image](https://user-images.githubusercontent.com/1477453/80131361-ca182e80-85a2-11ea-8cd3-9a6312fff646.png)